src/graph_notebook/seed/queries/propertygraph/__init__.py

```diff
@@ -1,4 +0,0 @@
-"""
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is there a reason we are removing this copyright?

Reply:
File was removed as part of the `.../seed/queries` directory. The queries folder and its contents have been relocated to our public production S3 bucket at `s3://aws-neptune-notebook/queries/`.